web/views.py

```diff
@@ -18,8 +18,9 @@ def about(request):
 	return render(request, 'pages/about.html',{})
 
 def members(request):
-	members_list = UserProfile.objects.all().filter(user__is_active=True)
-	return render(request, 'pages/members.html',{'members_list': members_list})
+	mentors_list = UserProfile.objects.all().filter(user__is_active=True).exclude(activity_status= 'SLEEPY').exclude(activity_status= 'LAZY').exclude(activity_status= 'ACTIVE') 
```

Comment:
You don't want `all()`, since you're filtering.

Also for `exclude()`, use `__in`, so `activity_status__in=('A', 'B',)` : https://docs.djangoproject.com/en/1.7/ref/models/querysets/#in